v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -176,4 +176,16 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   String getDLQDirectory();
 
   void setDLQDirectory(String value);
+
+  @TemplateParameter.GcsReadFile(
+      order = 15,
+      optional = true,
+      helpText =
+          "Transformation context file path in cloud storage used to populate data used in"
+              + " transformations performed during migrations. Eg: Maps db to shard id to"
+              + " identify the db from which shard a row was migrated",
+      description = "Transformation context file path in cloud storage")
+  String getTransformationContextFilePath();
```

Comment:
I'm assuming this is the shard configuration file. We should name it the same.

Reply:
shardConfig is used in reverse replication. For forward, we use this only.